x11/urw-base35-fonts/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                urw-base35-fonts
+version             2017-07-27
+set git_version     91edd6e
+categories          x11 fonts
+maintainers         {@lemzwerg gnu.org:wl} openmaintainer
+license             AGPL-3-with-embedded-font-exception
+platforms           darwin
+supported_archs     noarch
+
+description         URW base35 fonts in various formats
+long_description    "Recent Ghostscript versions come with a new set\
+                    of URW fonts that have additional support for\
+                    the Greek and Cyrillic scripts (thus superseding\
+                    the fonts contained in the 'urw-fonts' port).\
+                    Ghostscript distributes the URW fonts in Type1\
+                    format; in addition to these files, this port\
+                    also contains the corresponding Type1 metrics\
+                    files (AFM), together with TrueType (TTF) and\
+                    OpenType (OTF) versions of the fonts."
+
+homepage            https://www.ghostscript.com
+master_sites        "https://git.ghostscript.com/?p=urw-core35-fonts.git;a=snapshot;h=${git_version};sf=tgz;dummy="
```

Comment:
I have never seen `master_sites` quoted before. Please remove the quotes here too.

Reply:
Similar issue.  If you do `git grep 'master_sites.*;'` you can see that it is necessary to either use `\;` instead of `;` or quote the whole string.  I like the latter more.

Reply:
@lemzwerg Please don't resolve conversations on behalf of project members asking for changes. If @mf2k feels comfortable with your responses, he'll decide to accept.

Reply:
Ah, ok.  A misunderstanding.  I thought it is my job to press the 'resolve' button to indicate that I've sufficiently answered a question.